omniscidb/QueryEngine/NativeCodegen.cpp

```diff
@@ -1428,9 +1428,18 @@ Executor::compileWorkUnit(const std::vector<InputTableInfo>& query_infos,
 
   addTransientStringLiterals(ra_exe_unit, row_set_mem_owner);
 
+  bool row_func_not_inlined = false;
+  bool is_gpu_smem_used = false;
+
+  GPUTarget target{gpu_mgr, blockSize(), cgen_state_.get(), row_func_not_inlined};
```

Comment:
Do you think we could get rid of this structure at all?

Reply:
Not sure, it seems to be used in different place. If we want to remove it we should put it as a separate issue